src/model/Model.java

```diff
@@ -384,6 +387,15 @@ private void updateCachedList(List list, List newList, String repoId){
 	        @Override
 	        public void run() {
 	        	list.removeAll(removed);
+  	
+	        	Listable listItem = (Listable)newList.get(0);
+	        	if (listItem.getClass() == TurboMilestone.class) {
+	        		logNumOfUpdates(newList, "milestone(s)");
+	        	} else if (listItem.getClass() == TurboLabel.class) {
+	        		logNumOfUpdates(newList, "label(s)");
+	        	} else if (listItem.getClass() == TurboUser.class) {
+	        		logNumOfUpdates(newList, "collaborator(s)");
+	        	}
```

Comment:
Using `instanceof` is probably better, as it's less brittle (it'll still work if we subclass any of the `Turbo*` classes).